homeassistant/components/mopeka/config_flow.py

```diff
@@ -60,6 +61,28 @@ async def async_step_bluetooth_confirm(
             step_id="bluetooth_confirm", description_placeholders=placeholders
         )
 
+    async def async_step_medium_type(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Pick a MediumType."""
+        if user_input is not None:
+            assert self.unique_id is not None
+            return self.async_create_entry(
+                title=self._discovered_devices[self.unique_id],
+                data={MEDIUM_TYPE: user_input[MEDIUM_TYPE]},
+            )
+
+        return self.async_show_form(
+            step_id=MEDIUM_TYPE,
```

Comment:
Please use a different constant or bare string for the step since these are not the same